testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDatabricksShallowCloneTableCompatibility.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.tests.product.deltalake;
+
+import com.google.common.collect.ImmutableList;
+import io.trino.tempto.assertions.QueryAssert;
+import io.trino.testng.services.Flaky;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+import static io.trino.tempto.assertions.QueryAssert.Row.row;
+import static io.trino.tempto.assertions.QueryAssert.assertThat;
+import static io.trino.testing.TestingNames.randomNameSuffix;
+import static io.trino.tests.product.TestGroups.DELTA_LAKE_DATABRICKS;
+import static io.trino.tests.product.TestGroups.DELTA_LAKE_EXCLUDE_73;
+import static io.trino.tests.product.TestGroups.DELTA_LAKE_EXCLUDE_91;
+import static io.trino.tests.product.TestGroups.DELTA_LAKE_OSS;
+import static io.trino.tests.product.TestGroups.PROFILE_SPECIFIC_TESTS;
+import static io.trino.tests.product.deltalake.util.DeltaLakeTestUtils.DATABRICKS_COMMUNICATION_FAILURE_ISSUE;
+import static io.trino.tests.product.deltalake.util.DeltaLakeTestUtils.DATABRICKS_COMMUNICATION_FAILURE_MATCH;
+import static io.trino.tests.product.deltalake.util.DeltaLakeTestUtils.dropDeltaTableWithRetry;
+import static io.trino.tests.product.utils.QueryExecutors.onDelta;
+import static io.trino.tests.product.utils.QueryExecutors.onTrino;
+
+public class TestDeltaLakeDatabricksShallowCloneTableCompatibility
+        extends BaseTestDeltaLakeS3Storage
+{
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_OSS, DELTA_LAKE_EXCLUDE_73, DELTA_LAKE_EXCLUDE_91, PROFILE_SPECIFIC_TESTS})
+    @Flaky(issue = DATABRICKS_COMMUNICATION_FAILURE_ISSUE, match = DATABRICKS_COMMUNICATION_FAILURE_MATCH)
+    public void testShallowCloneSupport()
+    {
+        String baseTable = "test_dl_base_table_" + randomNameSuffix();
+        String shallowClonedTableV1 = "test_dl_shallow_clone_tableV1_" + randomNameSuffix();
+        String shallowClonedTableV2 = "test_dl_shallow_clone_tableV2_" + randomNameSuffix();
+
+        try {
+            // creating base table
+            String createTable = "CREATE TABLE default." + baseTable + " (a_number INT)" + " USING delta " +
+                    " LOCATION 's3://" + bucketName + "/databricks-compatibility-test-" + baseTable + "'" +
+                    " TBLPROPERTIES (" + " 'delta.minReaderVersion'='2'," + " 'delta.minWriterVersion'='5')";
```

Comment:
`SHALLOW CLONE` requires these reader & writer versions? Is it possible to avoid specifying those versions? Please avoid using redundant string concatenation if we need to put those properties explicitly.

Reply:
@ebyhr when i create a shallow cloned table, the protocol JSON has the following entries
```
{"protocol":{"minReaderVersion":1,"minWriterVersion":2}}
```
I suppose we need not specify them explicitly, will remove this

Reply:
@ebyhr handled redundant string concatenation